components/event-publisher-proxy/README.md

```diff
@@ -29,6 +29,26 @@ $ make test-local
 $ ko apply -f config/
 ```
 
+### Send Events
+
+```bash
+curl -v -X POST \
+    -H "Content-Type: application/cloudevents+json" \
+    --data @<(cat <<EOF
```

Comment:
why use file data here, and not just inline data? *cough* 'useless use of cat' *cough*